fbgrab

```diff
@@ -99,7 +101,14 @@ then
   chvt "$grab_vt"
 fi
 sleep "$wait_after_switch"
-fbcat "$device" | ppm2png "$1"
+
+if [ "$1" = "-" ]
```

Comment:
perhaps this if statement should be moved into `ppm2png()` function for `pnmtopng` since that is really the only case that needs to be modified.